lib/node_modules/@stdlib/array/empty-like/README.md

```diff
@@ -0,0 +1,150 @@
+<!--
+
+@license Apache-2.0
+
+Copyright (c) 2023 The Stdlib Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+-->
+
+# emptyLike
+
+> Create an uninitialized array having the same length and data type as a provided array.
+
+<!-- Section to include introductory text. Make sure to keep an empty line after the intro `section` element and another before the `/section` close. -->
+
+<section class="intro">
+
+</section>
+
+<!-- /.intro -->
+
+<!-- Package usage documentation. -->
+
+<section class="usage">
+
+## Usage
+
+```javascript
+var emptyLike = require( '@stdlib/array/empty-like' );
+```
+
+#### emptyLike( x\[, dtype] )
+
+Creates an uninitialized array having the same length and data type as a provided array `x`.
+
+```javascript
+var x = [ 1, 2, 3, 4, 5 ];
+
+var arr = emptyLike( x );
+// returns <Float64Array>
```

Comment:
@steff456 Why would the return array be `Float64Array` when the data type of the input array is "generic"?

Reply:
@Planeshifter Should this have passed linting? CI passed, which surprises me.

Reply:
This is the behavior we have in `empty`, if the dtype is not provided it will return a `Float64Array`.

Reply:
This function is different than `empty`. In `empty`, we are not provided an input array as a reference from which to infer `dtype` and `shape` info. See zeros-like, ones-like, and full-like for analogous APIs.

Reply:
@steff456 L51 still needs to be updated.